2996_reflection/reflection.md

```diff
@@ -3513,6 +3578,23 @@ Break the next paragraph into a bulleted list, extend it to also cover splices,
 
 :::
 
+### [expr.prim.lambda.capture]{.sref} Captures {-}
+
+Modify bullet 7.1 as follows:
+
+::: std
+- [7.1]{.pnum} An `$id-expression$` [or `$splice-expression$`]{.addu} that names a local entity potentially references that entity; an `$id-expression$` that names one or more non-static class members and does not form a pointer to member ([expr.unary.op]) potentially references `*this`.
```

Comment:
Should the second `$id-expression$` also get the "or `$splice-expression$`" treatment? If not, why not?

```cpp
struct C {
  int i;

  void f() {
    constexpr auto r = ^^i;
    [this]{
      f([:r:]); // is this... 
      f(i); // ... equivalent to this?
    };
  }
};
```

Reply:
No, because _splice-expression_ doesn't undergo the implicit `(*this)`-transformation; you must write `this->[:r:]`.